app/jobs/export_job.rb

```diff
@@ -72,6 +72,10 @@ def perform(user_id, export_type)
                             excel_export_histories(group),
                             build_filename('Sara-Alert-Full-Export-Histories', file_index, file_extension),
                             export_type)
+        lookups << get_file(user_id,
+                            excel_export_dosages(group),
+                            build_filename('Sara-Alert-Full-Export-Dosages', file_index, file_extension),
```

Comment:
In line of the other PR, did we want this (as well as the other export files) to be VACCs instead of Sara-Alert?

Reply:
I changed this in this PR https://github.com/SaraAlertVaccine/SaraAlert/pull/8